plugins/obs-webrtc/whip-output.h

```diff
@@ -19,6 +21,11 @@
 	blog(level, "[obs-webrtc] [whip_output: '%s'] " format, \
 	     obs_output_get_name(whipOutput->output), ##__VA_ARGS__)
 
+// maximum size for a video fragment, keep it under a std MTU of 1500
+// effective range is 576-1470, with the lower the value equating to
+// more packets created
+const uint16_t max_fragment_size(1180);
```

Comment:
I think `constexpr` would be better.
I'm afraid `the` might be redundant in `with the lower the value`  but I'd like to leave it to be commented by another one.

Reply:
I believe `with a lower value` would be a more correct way of phrasing it.

Reply:
Darn it, failing at my native language :)